tutormfe/patches/openedx-lms-development-settings

```diff
@@ -17,3 +17,29 @@ CORS_ORIGIN_WHITELIST.append("http://{{ MFE_HOST }}:{{ app["port"] }}")
 LOGIN_REDIRECT_WHITELIST.append("{{ MFE_HOST }}:{{ app["port"] }}")
 CSRF_TRUSTED_ORIGINS.append("{{ MFE_HOST }}:{{ app["port"] }}")
 {% endfor %}
+
+# Start of Dynamic config API settings
+
+ENABLE_MFE_CONFIG_API = True
+
+MFE_CONFIG = {
+    "BASE_URL": "{{ MFE_HOST }}",
+    "CREDENTIALS_BASE_URL": "/csrf/api/v1/token",
+    "DISCOVERY_API_BASE_UR": "{% if DISCOVERY_HOST is defined %}{% if ENABLE_HTTPS %}https{% else %}http{% endif %}://{{ DISCOVERY_HOST }}{% endif %}",
+    "ENABLE_NEW_RELIC": False,
+    "FAVICON_URL": "{% if ENABLE_HTTPS %}https{% else %}http{% endif %}://{{ LMS_HOST }}/favicon.ico",
```

Comment:
These settings will not work in development because in `tutor dev` the Caddy container is not running. 

- Get rid of the `ENABLE_HTTPS` conditionals: in development it is assumed everywhere that we run in http mode.
- Replace `{{ LMS_HOST }}` by `{{ LMS_HOST }}:8000`.
- Please do test your changes this in development.